### ISSUE TITLE
Seems to cut first and last character from tags

### ISSUE THREAD

**Issue by isakrubin** [SamHH/bukubrow#64]
Version 2.4.0, on Chrome Version 69.0.3497.100 (Official Build) (64-bit)

When creating bookmarks with multiple tags it seems to remove first char from the first tag and last char from the last tag.

Is this a known issue?

Example:

```
buku -p
.....
1091. Censored
   > https://censored
   + censored
   # apple,banana,test

1092. Censored
   > https://censored
   + censored
   # pple,banana,tes

1093. Censored
   > http://censored
   + censored
   # pple,banana,tes

```
if I then open it for editing it looks correct, and after a save it also looks correct when listing.
```
EDITOR=vim buku -w 1092
buku -p
.....
1092. Censored
   > https://censored
   + censored
   # apple,banana,test
```

**Comment by SamHH** [comment]
Good find!

It must be related to how exactly I'm serialising the array back into the Buku database. I do vaguely recall that Buku's database left some stray commas around, it may be related to that.

I'll investigate it this weekend and see whether it's something to be fixed here or over in the Buku repo. :+1:

**Comment by isakrubin** [comment]
"This has been fixed in 2.4.2." Is 2.4.2 available anywhere? seems like last release was back in september.

**Comment by SamHH** [comment]
2.4.2 is available on both the Firefox and Chrome addon stores now; sometimes there's a little delay before they're published, particularly in the former case.

I tend now to only cut GitHub releases when there are binary changes as I don't supply the extensions as a third-party (if you wanted them that way, you'd probably prefer to just clone/pull the repo and build yourself).
